libs/core/src/components/pds-radio/pds-radio.scss

```diff
@@ -107,6 +109,10 @@ label {
   margin-inline-start: 10px;
 }
 
+.pds-radio__container {
+  display: flex;
```

Comment:
With the addition of the wrapping `pds-radio__container`, can we remove the flex styling from the host and apply to the wrapper since it will no longer be needed? (lines 9-11)